accord-core/src/main/java/accord/local/Cleanup.java

```diff
@@ -134,9 +138,12 @@ public static Cleanup shouldCleanup(TxnId txnId, Status status, Durability durab
                 return Cleanup.NO;
             case SHARD_REDUNDANT:
                 if (enforceInvariants && status.hasBeen(PreCommitted) && !status.hasBeen(Applied) && redundantBefore.preBootstrapOrStale(txnId, toEpoch, route.participants()) != FULLY)
-                    illegalState("Loading redundant command that has been PreCommitted but not Applied");
+                    illegalState("Loading redundant command that has been PreCommitted but not Applied.");
         }
 
+        if (!status.hasBeen(PreCommitted))
```

Comment:
This is simply clarifying the cases where we know the command has been Invalidated (rather than we don't know if it was Invalidated or Applied and then Erased)